src/harness/simple_crl_server.py

```diff
@@ -0,0 +1,105 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+    A implementation of Certificate Revocation list Server.
+"""
+import logging
+import os
+import threading
+import urlparse
+from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
+
+
+class CrlServerTestHarness(threading.Thread):
+  """
```

Comment:
Docstrings should not begin with a newline.  See https://www.python.org/dev/peps/pep-0257/

Reply:
Changed   accordingly